public/balloon.js

```diff
@@ -1,27 +1,29 @@
-document.getElementById('strange').addEventListener('click', function(){
-  document.getElementById('strange').classList.add('pop');
-});
+if (document.querySelector('.personality')) {
```

Comment:
See here: https://github.com/CYPIAPT-LNDSE/welcome-to-camhs/issues/58

This file has lots of repetition .. Do you need advice on how to make this better? Its one of the things I said that needed fixing.

I know you have only changed the indentation here, but the plan was to respond to my comments before using this code.